docs/source/author/topics.rst

```diff
@@ -64,14 +70,17 @@ For more information about what this page displays, see :doc:`../developer/dev`.
 Topics Content Directory
 ==============================================================================
 
-The ``content`` directory for the topics application contains
+The ``content`` directory for the topics application contains:
 
-- a directory for each language in which content exists, named using the Django locale code. This directory contains
+- A directory for each language in which content exists, named using the Django
+  locale code. These language directories each contain:
 
-  - Content Markdown files
-  - YAML files containing translatable strings
+  - A folder for each Topic, containing the content Markdown files.
+  - YAML files containing strings that need to be translated in each different language folder.
 
-- a special ``structure`` directory which contains all configuration YAML files
+- Several auto-generated directories. Please ignore these.
+
+- A ``structure`` directory which contains all configuration YAML files.
```

Comment:
Real nit picky but can we switch line 81 with 83 so that the least relevant info appears last? :stuck_out_tongue: